rust/examples/Cargo.toml

```diff
@@ -25,4 +25,5 @@ shellexpand = { workspace = true }
 borsh = { workspace = true }
 phoenix-v1 = { workspace = true }
 phoenix-sdk = { path = "../phoenix-sdk" }
-devnet-token-faucet = { git = "https://github.com/Ellipsis-Labs/devnet-token-faucet" }
\ No newline at end of file
+devnet-token-faucet = { git = "https://github.com/Ellipsis-Labs/devnet-token-faucet" }
```

Comment:
should we publish this? cc @jarry-xiao

Reply:
We already published an older version (with a dep conflict) so it feels fine to publish the new one?

Reply:
I think we can rename it to something like generic-token-faucet. Might be worth a contract update so you can't mint u64::MAX at a time permissionlessly though! Don't want to get griefed on devnet

Reply:
Update: upgraded the program including the new mint cap and published a new crate under `generic-token-faucet`. Latest commit [28f1f7a](https://github.com/Ellipsis-Labs/phoenix-sdk/pull/87/commits/28f1f7ae072e717c905de3cb0c5581e10a0bb3f4) has the relevant renaming fixes to this PR